internal/config/supervisor/types.go

```diff
@@ -16,6 +16,15 @@ type Config struct {
 	Endpoints               *Endpoints        `json:"endpoints"`
 	AggregatedAPIServerPort *int64            `json:"aggregatedAPIServerPort"`
 	TLS                     TLSSpec           `json:"tls"`
+	Audit                   AuditSpec         `json:"audit"`
+}
+
+type AuditInternalPaths string
+
+const AuditInternalPathsEnabled = "Enabled"
+
+type AuditSpec struct {
```

Comment:
It looks like this configuration option is not exposed to the user yet. Also, we don't have a configuration option to prevent PII in the logs yet. Feels like we need at least the latter one before merging this branch, right?

Reply:
This is done now!